test/qa/lib/meshnet.py

```diff
@@ -20,27 +20,141 @@
 strip_colors = re.compile(r'\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])', flags=re.IGNORECASE)
 
 
+class Peer:
+    def __init__(
+            self,
+            hostname: str,
+            nickname: str,
+            ip: str,
+            public_key: str,
+            os: str,
+            distribution: str,
+            status: str = None,
+            allow_incoming_traffic: bool = None,
+            allow_routing: bool = None,
+            allow_lan_access: bool = None,
+            allow_sending_files: bool = None,
+            allows_incoming_traffic: bool = None,
+            allows_routing: bool = None,
+            allows_lan_access: bool = None,
+            allows_sending_files: bool = None,
+            accept_fileshare_automatically: bool = None
+            ):
+        self.hostname = hostname
+        self.nickname = nickname
+        self.status = status
+        self.ip = ip
+        self.public_key = public_key
+        self.os = os
+        self.distribution = distribution
+        self.allow_incoming_traffic = self._convert_to_bool(allow_incoming_traffic)
+        self.allow_routing = self._convert_to_bool(allow_routing)
+        self.allow_lan_access = self._convert_to_bool(allow_lan_access)
+        self.allow_sending_files = self._convert_to_bool(allow_sending_files)
+        self.allows_incoming_traffic = self._convert_to_bool(allows_incoming_traffic)
+        self.allows_routing = self._convert_to_bool(allows_routing)
+        self.allows_lan_access = self._convert_to_bool(allows_lan_access)
+        self.allows_sending_files = self._convert_to_bool(allows_sending_files)
+        self.accept_fileshare_automatically = self._convert_to_bool(accept_fileshare_automatically)
+
+    def _convert_to_bool(self, value):
+        return value.lower() == "enabled" if value is not None else None
+
+
 class PeerName(Enum):
     Hostname = 0
     Ip = 1
     Pubkey = 2
 
 
+class PeerList:
+    def __init__(self):
+        self.this_device: list[Peer] = []
+        self.internal_peers: list[Peer] = []
+        self.external_peers: list[Peer] = []
+
+    def _str_to_dictionary(self, data: str):
+        new_dictionary = {a.strip().lower(): b.strip()
+                for a, b in (element.split(':')
+                                for element in
+                                filter(lambda line: len(line.split(':')) == 2, data.split('\n')))}
```

Comment:
It is not readable what is going on here. Maybe split it into two separate generators?